blocks.md

```diff
@@ -62,17 +62,21 @@ critical_updates_regex = "(linux |linux-lts|linux-zen)"
 Key | Values | Required | Default
 ----|--------|----------|--------
 `interval` | Update interval in seconds. | No | `600`
-`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{count}"`
-`format_singular` | Same as `format`, but for when exactly one update is available. | No | `"{count}"`
-`format_up_to_date` | Same as `format`, but for when no updates are available. | No | `"{count}"`
+`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{count:1}"`
+`format_singular` | Same as `format`, but for when exactly one update is available. | No | `"{count:1}"`
+`format_up_to_date` | Same as `format`, but for when no updates are available. | No | `"{count:1}"`
 `warning_updates_regex` | Display block as warning if updates matching regex are available. | No | `None`
 `critical_updates_regex` | Display block as critical if updates matching regex are available. | No | `None`
 
 #### Available Format Keys
 
```

Comment:
Or perhaps it could be added here. For every block that has format strings, to increase visibility of the feature.